apps/mobile-wallet/src/screens/new-wallet/AddBiometricsScreen.tsx

```diff
@@ -107,6 +107,7 @@ const AnimationContainer = styled.View`
 
 const StyledAnimation = styled(LottieView)`
   width: 300px;
+  height: 120%;
```

Comment:
nit: It's very strange to see a value larger than `100%`! Is this the right way to fix the issue?